app/helpers/home_helper.rb

```
@@ -49,7 +49,7 @@ def indicator_badge_status(count, total)
         :error
       when 0.51..0.99
         :warning
-      when 1
+      else
```

du coup ça ne peut être que 1 si tu as scopé à `active_students`?

---

Yes, mais je pense que le compteur devrait pas planter si au dessus de 1 en vrai. C'est pas sa responsabilité de checker les ratios. Je préfère afficher un ratio à 104/102 et que les users nous contactent plutôt que de leur bloquer l'appli

---

pas d'accord ; je suis adepte du fail early et ici il n'y aucune raison que ce ratio soit au dela de 1. Ça nous a d'ailleurs permis de corriger la mauvaise requête qui n'utilisaient pas les bons étudiants, donc raison de plus pour garder le circuit "normal". Je préfère largement que ça plante vite et qu'on corrige vite plutôt qu'avoir une application dans un état "étrange" qui affiche 104 ribs sur 102 🙃

---

All right